### DISCUSSION TITLE
Phone is being remotely managed help!

### DISCUSSION THREAD

**Discussion by Gotem007** [community/community#38587] [Mobile]
So I’m needing help with getting these off my home and keeping them off.. my phone is being sent thousands of packets and that’s just been since this morning. It’s 10 o’clock now… There is a hidden at work where I live and I have already found one raspberry pi so Im pretty sure that  has something to do with what is sending the packets.. i’m like trying to take a crash course in Coding but I’m not able to learn fast enough, so could y'all please help out?

**Comment by cx0y** [comment]
May be your phone has been attacked by some kind of payload or backdoor! is it working automatically? for now disconnect network from you phone! then check what happening!

**Reply by Gotem007** [reply]
Yeah it’s automatic I’ve reset my phone to factory couple of times and it comes right back. This is also my second phone that I’ve had this problem with. It’s actually been going on for about a year and a half to two years… so I have no clue what to do. Wish there was a way to try to find out who it is… Because it’s gotta be somebody I know because of the fact that it’s gone on for so long and also started on my new phone with a different cellular plan and number. I didn’t log into any accounts that I had on my other phone either. 
i’m getting tired of having to reset my phone and start all over again.

**Reply by Gotem007** [reply]
![4E5D38FF-B446-446B-A299-61B400FD744D](https://user-images.githubusercontent.com/117626203/200662938-e320562a-52d9-4903-bb38-8b8d6d91f92e.png)
![CA707313-2E6D-4717-9B11-6DE8AEBD6C69](https://user-images.githubusercontent.com/117626203/200662955-4fc5c730-92a0-4833-8e53-f8ffbf028f96.png)
![169BB577-00DC-402C-9EB1-7CC2172D1039](https://user-images.githubusercontent.com/117626203/200662963-cfa7051b-0c68-4903-a537-465af5cea95b.png)
![3AA9348F-6677-4AEC-980F-968F05F7F903](https://user-images.githubusercontent.com/117626203/200662971-5d6cb5f0-ad57-46c1-b4c2-c00d2a530f60.png)
![1F1C42BD-3E72-44CA-BE4C-7AE695755129](https://user-images.githubusercontent.com/117626203/200662973-018e7d4f-b947-470d-9cc5-4da1a79f2f82.png)
![AC9542C4-1BB7-423D-AFD7-35AF3F835800](https://user-images.githubusercontent.com/117626203/200662978-72c63589-b7c3-4a42-8fe6-9617162d6bf8.png)

**Reply by Gotem007** [reply]
They had errors because I was deleting them. But they’re back and they’re loading Xcode, as well as JavaScript..

**Comment by cx0y** [comment]
If they are using several proxy you can't easily find out who they are or is it. ok! first things first, you need to activate 2 factor authentication each of your accounts. Use a password manager, `Bitwarden` or `Norton` is good. Change your all accounts passwords (use a password generator) and never save any password in your device. Payloads can access you camera or microphone, so careful! try to find out, which app has suspicious permission (such as camera, microphone, files, location, contacts etc..), also it is possible, can attacked by pdfs, image files, excel files and I'm thinking about this problem, it might take some time. I'll notify you, if I can help!

**Reply by SRAGEIN** [reply]
- @github-staff

**Reply by cx0y** [reply]
> * @github-staff

🙂

**Comment by ConvincibleViln** [comment]
I need help myself I am not able to do it it's an emergency need help urgent long story

**Comment by ConvincibleViln** [comment]
Dude please contact someone that will come here in person I can't do it myself 
121 oak trail Belmont nc 28012 share my story until we win this battle. I'm outgunned by myself severely!!

**Comment by ConvincibleViln** [comment]
thanks

**Comment by Jonnypleasehelp** [comment]
I have had this happen to me too
